src/main/java/uk/gov/companieshouse/overseasentitiesapi/utils/NationalityOtherMapping.java

```diff
@@ -14,4 +14,12 @@ public static String generateNationalityOtherField(String nationality, String se
             return String.format("%s, %s", nationality, secondNationality);
         }
     }
+
+    public static String generateMoNationalityOtherField(String nationality, String secondNationality) {
```

Comment:
Maybe rename this generateNationalityOtherChangeField? 
The BO changes have a different way of doing it. Can leave but want to be more clear that adds use ", " & changes just "".